test/integration/toolchains/gnu/test_makedeps.py

```diff
@@ -413,7 +413,7 @@ def package_info(self):
 
 @pytest.mark.parametrize("pattern, result, expected",
                          [("libs = []", False, 'SYSROOT'),
-                          ("sysroot = ['/foo/bar/sysroot']", True, 'CONAN_SYSROOT_PACKAGE = /foo/bar/sysroot')])
```

Comment:
Should we not adapt the code for Python 3.13 then? I mean, if that's the expected result, then we should make it backward compatible to behave in the same way, right? wdyt?